pkg/capabilities/cli/cmd/generated_info.go

```diff
@@ -0,0 +1,173 @@
+package cmd
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"reflect"
+	"strings"
+	"unicode"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+)
+
+type GeneratedInfo struct {
+	Package        string
+	Config         Struct
+	Input          *Struct
+	Types          map[string]Struct
```

Comment:
`Struct`	is a confusing and non obvious name. can we pick something better?

Reply:
It's pretty common among generators to represent the info about the struct that you are creating in a type called Struct.  Do you have a better name in mind?